spec.md

```diff
@@ -901,7 +901,12 @@ wire mywire: UInt
 
 ## Registers
 
-A register is a named stateful circuit component.
+A register is a named stateful circuit component.  Reads from a register return 
+the value of the element, writes are not visible until after the positive edges 
```

Comment:
I would emphasize that read return the **current** value of the element.

Writes are not visible until after **a positive edge on** the register's clock port.